[2024-08-19T23:24:54.360+0100] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-19T23:24:54.392+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: jobs_etl.Loading_Data_to_DataBase scheduled__2024-08-18T18:00:00+00:00 [queued]>
[2024-08-19T23:24:54.401+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: jobs_etl.Loading_Data_to_DataBase scheduled__2024-08-18T18:00:00+00:00 [queued]>
[2024-08-19T23:24:54.401+0100] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-19T23:24:54.413+0100] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Loading_Data_to_DataBase> on 2024-08-18 18:00:00+00:00
[2024-08-19T23:24:54.425+0100] {standard_task_runner.py:63} INFO - Started process 85085 to run task
[2024-08-19T23:24:54.431+0100] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'jobs_etl', 'Loading_Data_to_DataBase', 'scheduled__2024-08-18T18:00:00+00:00', '--job-id', '410', '--raw', '--subdir', 'DAGS_FOLDER/jobs_etl.py', '--cfg-path', '/tmp/tmpugia0gnd']
[2024-08-19T23:24:54.433+0100] {standard_task_runner.py:91} INFO - Job 410: Subtask Loading_Data_to_DataBase
[2024-08-19T23:24:54.507+0100] {task_command.py:426} INFO - Running <TaskInstance: jobs_etl.Loading_Data_to_DataBase scheduled__2024-08-18T18:00:00+00:00 [running]> on host huser
[2024-08-19T23:24:54.626+0100] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='hassounibarka@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='jobs_etl' AIRFLOW_CTX_TASK_ID='Loading_Data_to_DataBase' AIRFLOW_CTX_EXECUTION_DATE='2024-08-18T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-18T18:00:00+00:00'
[2024-08-19T23:24:54.627+0100] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-19T23:24:54.673+0100] {logging_mixin.py:188} INFO - DB connected successfully
[2024-08-19T23:24:54.864+0100] {logging_mixin.py:188} INFO - Data inserted successfully from CSV
[2024-08-19T23:24:54.864+0100] {python.py:237} INFO - Done. Returned value was: None
[2024-08-19T23:24:54.865+0100] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-19T23:24:54.880+0100] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=jobs_etl, task_id=Loading_Data_to_DataBase, run_id=scheduled__2024-08-18T18:00:00+00:00, execution_date=20240818T180000, start_date=20240819T222454, end_date=20240819T222454
[2024-08-19T23:24:54.904+0100] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-19T23:24:54.929+0100] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-19T23:24:54.931+0100] {local_task_job_runner.py:222} INFO - ::endgroup::
